### PR TITLE
[DATASTD-1577] Add values to PerformanceLevel default values

### PR DESCRIPTION
**Pull request by ggaronzik**

### CODE REVIEW

File: Descriptors/PerformanceLevelDescriptor.xml [hunk 1]
```diff
@@ -19,63 +19,69 @@
...
+		<ShortDescription>Did Not Meet Standard</ShortDescription>
+		<Description>Did Not Meet Standard</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Well Below Basic</CodeValue>
-		<ShortDescription>Well Below Basic</ShortDescription>
-		<Description>Well Below Basic</Description>
+		<CodeValue>Pass</CodeValue>
+		<ShortDescription>DEPRECATE: Pass</ShortDescription>
```
Comment (blmeyers) [inline Descriptors/PerformanceLevelDescriptor.xml:41]: Ticket comment mentions "Keep Pass/Fail" but this is deprecating them, did the final decision on this change?

File: Descriptors/PerformanceLevelDescriptor.xml [hunk 2]
```diff
@@ -19,63 +19,69 @@
...
+		<ShortDescription>DEPRECATE: Commended Performance</ShortDescription>
+		<Description>DEPRECATE: Commended Performance</Description>
 		<Namespace>uri://ed-fi.org/PerformanceLevelDescriptor</Namespace>
 	</PerformanceLevelDescriptor>
 	<PerformanceLevelDescriptor>
-		<CodeValue>Commended Performance</CodeValue>
-		<ShortDescription>Commended Performance</ShortDescription>
-		<Description>Commended Performance</Description>
+		<CodeValue>Above Benchmark</CodeValue>
+		<ShortDescription>DEPRECATE: Above Benchmark</ShortDescription>
```
Comment (blmeyers) [inline Descriptors/PerformanceLevelDescriptor.xml:59]: Shouldn't all the DEPRECATE markers here be DEPRECATED?

**Review by blmeyers — CHANGES_REQUESTED** [review]

File: Descriptors/PerformanceLevelDescriptor.xml (hunk 2 above)
Comment (ggaronzik) [reply Descriptors/PerformanceLevelDescriptor.xml]: Fixed.

File: Descriptors/PerformanceLevelDescriptor.xml (hunk 1 above)
Comment (ggaronzik) [reply Descriptors/PerformanceLevelDescriptor.xml]: @ejansson, did we adjust the requirements here or should Pass/Fail be included? I don't see any notes in the ticket so I may have just been overzealous in deprecating those values.

**Review by blmeyers — APPROVED** [review]

File: Descriptors/PerformanceLevelDescriptor.xml (hunk 1 above)
Comment (blmeyers) [reply Descriptors/PerformanceLevelDescriptor.xml]: Going ahead and approving this, but we should either adjust the ticket or this change to match.

Comment (ggaronzik) [reply Descriptors/PerformanceLevelDescriptor.xml]: removed the pass/fail deprecated designation to align with the ticket.

@blmeyers, @ejansson

**Review by ejansson — APPROVED** [review]
Looks correct.

**Pull request merged**